docs/source/user-guide/tabular/supported_models.rst

```diff
@@ -1,107 +1,111 @@
 .. _supported_models:
 
-====================
-Supported Models
-====================
+**************************************
+Supported Models and Static Prediction
+**************************************
 
-Many checks require passing a model object. These include all model-evaluation related checks,
-and in general any check that requires the model's predictions for its analysis.
+Some checks, including all the model evaluation related checks, require model predictions in order to run.
+The prediction can be provided **either** within a model object or as pre-computed static predictions.
 
-.. Note::
-    In order to be compatible with deepchecks, the model object should adhere to one requirement:
-
-    - It has to have a a method enabling **prediction**
-
-More specifically, the model should have a ``predict`` method for regression tasks, 
-and for classification tasks also a ``predict_proba`` method, 
-which should be implemented using the scikit-learn api conventions. 
-Some checks may attempt using additional model methods if those exist, 
-for more information see :ref:`Optional Model Interface <supported_models__optional_model_interface>`.
+Whether provided from a model interface or as a pre-computed static predictions,
+the predictions must be in the following format based on the task type:
 
-Note that built-in scikit-learn classifiers and regressors, 
-along with many additional popular models types (e.g. XGBoost, LightGBM, CatBoost etc.) implement these methods 
-and are thus supported.
+* Classification: ``proba``, an `ndarray` of shape ``(n_samples, n_classes)`` containing the predicted
+  probability of each class per sample in the dataset.
+* Regression: ``pred``, an `ndarray` of shape ``(n_samples,)``, containing the predicted value
+  for each sample in the dataset.
 
+For classification tasks, it is optional to also supply ``pred`` predictions. If not supplied, Deepchecks will
+assume the predicted value is the class with the highest probability.
 
-Required Model Interface
-==========================
+Using a Model Interface
```

Comment:
My main comment here (many of the others are small...)
is that I don't feel it's clear for a user "which models are supported?"
and "how can I make my model supported if it isn't already?" (e.g. how to copy paste simple code for model wrapper, wrap their model given their task type, and return that object...)